src/components/button/button.scss

```diff
@@ -59,6 +78,15 @@
       width: 56px;
       padding: 18px;
     }
+    &.show-on-scroll {
+      display: none;
+    }
+    &.btn-float {
+      box-shadow: var(--kd-elevation-level-3);
+    }
+    &.btn-hidden {
+      display: none;
+    }
```

Comment:
Probably don't need to duplicate this CSS across the three sizes. Should only be needed once. 

It might be nice to give this a transition as well. Ex: Instead of `display none;`, transition from `visibility: hidden; opacity 0;` to `visibility: visible; opacity: 1;`

Reply:
Addressed.